spec/system/jobseekers_can_create_a_job_alert_from_a_search_spec.rb

```diff
@@ -3,23 +3,30 @@
 RSpec.describe "Jobseekers can create a job alert from a search" do
   let(:location) { nil }
   let(:location_category_search?) { false }
+  let(:jobseeker_accounts_enabled?) { false }
+  let(:jobseeker_signed_in?) { false }
+  let(:jobseeker) { build_stubbed(:jobseeker) }
 
-  context "when recaptcha score is invalid" do
-    before do
-      allow_any_instance_of(ApplicationController).to receive(:verify_recaptcha).and_return(true)
-      allow_any_instance_of(ApplicationController).to receive(:recaptcha_reply).and_return({ "score" => 0.1 })
-    end
+  before { allow(JobseekerAccountsFeature).to receive(:enabled?).and_return(jobseeker_accounts_enabled?) }
+
+  describe "#recaptcha" do
```

Comment:
`#asdf` normally refers to a method called `asdf`. I think remove the `#` to avoid confusion.